docs/management/connectors/action-types/inference.asciidoc

```diff
@@ -0,0 +1,123 @@
+[[inference-action-type]]
+== {infer-cap} connector and action
+++++
+<titleabbrev>{inference}</titleabbrev>
+++++
+:frontmatter-description: Add a connector that can send requests to {inference}.
+:frontmatter-tags-products: [kibana] 
+:frontmatter-tags-content-type: [how-to] 
+:frontmatter-tags-user-goals: [configure]
+
+
+The {infer} connector uses {es} client to send requests to {infer}. The connector uses the <<execute-connector-api,run connector API>> to send the request.
+
+[float]
+[[define-inference-ui]]
+=== Create connectors in {kib}
+
+You can create connectors in *{stack-manage-app} > {connectors-ui}*.  For example:
+
+[role="screenshot"]
+image::management/connectors/images/inference-connector.png[{inference} connector]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
```

Comment:
Is it really auto-generated or is this string a copy/paste error?

Reply:
copy/paste